.github/workflows/renovate.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
       - name: Self-hosted Renovate
-        uses: renovatebot/github-action@a6e57359b32af9a54d5b3b6603011f50629a0a05 # v40.1.2
+        uses: renovatebot/github-action@78bdcb3bffa5e95e646183ca0a2ac2895abd6a20 # v40.1.3
```

Comment:
maybe we don't want to add this change in the same PR ?

Reply:
I didn’t make this change intentionally , I assumed it is part of the project automation. Should I revert this commit then?

Reply:
Oh it's automation...why it worked... 😮 
It's merged in https://github.com/argoproj/argo-helm/pull/2579, so please fetch remote branch.